src/Land/Model/LandModel.jl

```diff
@@ -0,0 +1,302 @@
+    source::SRC
+end
+
+"""
+    vars_state_conservative(land::LandModel, FT)
+
+Conserved state variables (Prognostic Variables)
+"""
+function vars_state_conservative(land::LandModel, FT)
+    @vars begin
+        soil::vars_state_conservative(land.soil, FT)
+    end
+end
+
+"""
+    vars_state_auxiliary(land::LandModel, FT)
+
+Names of variables required for the balance law that aren't related to 
+derivatives of the state variables (e.g. spatial coordinates or various 
+integrals) or those needed to solve expensive auxiliary equations 
+(e.g., temperature via a non-linear equation solve)
+"""
+function vars_state_auxiliary(land::LandModel, FT)
+    @vars begin
+        soil::vars_state_auxiliary(land.soil, FT)
+    end
+end
+
+"""
+    vars_state_gradient(land::LandModel, FT)
+
+Names of the gradients of functions of the conservative state 
+variables.
+
+Used to represent values before **and** after differentiation
+"""
+function vars_state_gradient(land::LandModel, FT)
+    @vars begin
+        soil::vars_state_gradient(land.soil, FT)
+    end
+end
+
+"""
+    vars_state_gradient_flux(land::LandModel, FT)
+
+Names of the gradient fluxes necessary to impose Neumann boundary 
+conditions
+"""
+function vars_state_gradient_flux(land::LandModel, FT)
+    @vars begin
+        soil::vars_state_gradient_flux(land.soil, FT)
+    end
+end
+
+
+"""
+    flux_first_order!(
+        Land::LandModel,
+        flux::Grad,
+        state::Vars,
+        aux::Vars,
+        t::Real
+    )
+
+Computes and assembles non-diffusive fluxes in the model equations.
+"""
+function flux_first_order!(
+    land::LandModel,
+    flux::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+) end
+
+
+"""
+    compute_gradient_argument!(
+        land::LandModel,
+        transform::Vars,
+        state::Vars,
+        aux::Vars,
+        t::Real,
+    )
+
+Specify how to compute the arguments to the gradients.
+"""
+function compute_gradient_argument!(
+    land::LandModel,
+    transform::Vars,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+
+    compute_gradient_argument!(land, land.soil, transform, state, aux, t)
+end
+
+"""
+    compute_gradient_flux!(
+        land::LandModel,
+        diffusive::Vars,
+        ∇transform::Grad,
+        state::Vars,
+        aux::Vars,
+        t::Real,
+    )
+
+Specify how to compute gradient fluxes.
+"""
+function compute_gradient_flux!(
+    land::LandModel,
+    diffusive::Vars,
+    ∇transform::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+)
+
+    compute_gradient_flux!(
+        land,
+        land.soil,
+        diffusive,
+        ∇transform,
+        state,
+        aux,
+        t,
+    )
+
+end
+
+"""
+    flux_second_order!(
+        land::LandModel,
+        flux::Grad,
+        state::Vars,
+        diffusive::Vars,
+        hyperdiffusive::Vars,
+        aux::Vars,
+        t::Real,
+    )
+
+Specify F_{second_order} for each conservative state variable
```

Comment:
```suggestion
Specify ``F_{\mathrm{second_order}}`` for each conservative state variable
```
I think this is right?

Reply:
I think I'll go with ``F_{\mbox{second order}}`` so that there is a space (that looks right in latex editor). but yeah, ill make sure it looks ok! That second_order makes the o a subscript, and \mathrm{second order} removes the space.

Reply:
ah, well both seem to be giving me "ERROR: LoadError: LoadError: syntax: invalid escape sequence" in the documentation build.

Reply:
just going to write it out as second order flux :)